frameworks/Rust/actix-4/src/models.rs

```diff
@@ -0,0 +1,126 @@
+use std::fmt::Display;
+
+use serde::{Deserialize, Serialize};
+
+pub const CONNECTION_POOL_SIZE: usize = 40;
+
+pub type Result<T> = std::result::Result<T, Error>;
+
+#[derive(Debug)]
+pub struct Error {
+    err: anyhow::Error,
+}
+
+impl Display for Error {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        self.err.fmt(f)
+    }
+}
+
+impl actix_web::error::ResponseError for Error {}
+
+impl<T> From<T> for Error
+where
+    T: Into<anyhow::Error>,
+{
+    fn from(e: T) -> Self {
+        Error { err: e.into() }
+    }
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct Message {
+    pub message: &'static str,
+}
+
+#[derive(Serialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct World {
+    pub id: i32,
+    pub random_number: i32,
+}
+
+// The ids are stored in MongoDB as floating point numbers, so need a custom deserialization implementation
```

Comment:
I plan to file an issue about this. I didn't just fix it now since I figured it might break loads of other existing tests. The issue is that the database is populated via the shell like `db.coll.insertOne({ id: 5 })`, which the shell interprets as a float by default.

Reply:
Maybe rather than two distinct `Deserialize` impls for `World` and `Fortune`, make an `I32asF32` type or some such?

Reply:
I tried this out, but then all the places that use `World` and `Fortune` needed to be updated to handle the newtype, so it just seemed easier to leave it as-is.